hawkular-alerts-metrics/src/main/java/org/hawkular/alerts/external/metrics/Expression.java

```diff
@@ -58,44 +60,22 @@
         min,
         /** Maximum reported value for the period. */
         max,
-        // Group metric only
-        /** number of distinct [trait] values for the group. No period. */
-        card(false),
-        /** percent [0.0,1.0] with last-reported availability down. No period */
-        down(false),
-        /** percent [0.0,1.0] with last-reported availability up. No period */
-        up(false);
-
-        private final boolean single; // true if supported by single metric/ requires a period
+        /** Count of UP avail for the period. */
+        heartbeat;
```

Comment:
Yes, this is a new feature for the metrics external alerter.  It is something I discussed with Stefan who was wondering whether we could do something like this.  See here, this is the updated external alerter doc for metrics:

https://docs.google.com/document/d/1vKV7ddPo4-LnHfP35jPoV0kSTLbhicCl1goldRzfSQA/edit#heading=h.kk5oqu4purxl